### ISSUE TITLE
Discover the status of discuss.openai.com in 2019

### ISSUE THREAD

**Issue by fititnt** [fititnt/ais-ethics-open-groups#1]
**TL;DR: discover if discuss.openai.com is temporary or permanent disabled; if the second case, look for some official explanation**

---

The <https://discuss.openai.com/> is (or was) one of the places open on the internet to discuss philosophy and A/IS ethics.

I remember that people around the world were discussing topics that were not just about how to develop AI, but about morality, risks, impact, etc, in one of the sub forums. And some of these threads where big.

At this moment, the adress "discuss.openai.com" does not resolve to a IP address. I tested from different IPs and proxies, and still not able to connect.

Looking from cached pages of the site, I'm able to find that **at least until 2018-12-23 the discuss.openai.com was online**.

![captura de tela de 2019-01-29 02-15-45](https://user-images.githubusercontent.com/812299/51883827-ced43f80-236b-11e9-930b-7205dc34f993.png)

---

![captura de tela de 2019 01 29 02 18 50](https://user-images.githubusercontent.com/812299/51883963-68035600-236c-11e9-8c91-63d2d0bc9d8e.png)

**Comment by fititnt** [comment]
Asked as a question at https://ai.stackexchange.com/questions/10282/status-of-discussion-forum-discuss-openai-com-in-2019.

**Comment by fititnt** [comment]
For reference, please look at https://ai.stackexchange.com/questions/10282/status-of-discussion-forum-discuss-openai-com-in-2019. But I guess is bad news.

Maybe I will open a different topic on the AI stack exchange on how to handle shutdown of AI forums. Draft here https://github.com/EticaAI/forum/issues/65#issuecomment-461253123.

But for this issue, I will close and remove the OpenAI forum.
